src/test/java/study/StringCalculator.java

```diff
@@ -0,0 +1,97 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringCalculator {
+
+    int calculator(String str) {
+        String[] cal_element = str.split(" ");
```

Comment:
Java의 경우 Code Convention은 Camel Case로 사용합니다. 아래 내용을 한번 참고해보세요

https://naver.github.io/hackday-conventions-java/#var-lower-camelcase

```suggestion
        String[] calElement = str.split(" ");
```

Reply:
좋은 페이지 공유 감사합니다! 정독하고 고치겠습니다!